docs/explanations/introduction.md

```diff
@@ -0,0 +1,29 @@
+
+## Checkpointing and 51% attacks
+
+51% attacks are possible when an adversary has more than half of the hashing power in the entire network, meaning they can produce blocks at a faster rate than all other miners combined. Over time this allows them to create a forking chain in private which is arbitrarily longer than the longest public chain. At any point the attacker can then disclose their private chain to the public, at which point everybody will switch to the attackers chain, since it's longer. This is a problem because the attacker can choose which transactions are included in their chain. As an example, double spending is possible by not including an already considered confirmed transaction from the public chain in their private chain.
+
+An important aspect to consider is that 51% attacks become easier with lower mining participation, since an attacker only needs more mining power than the entirety of the lawful miners. Because of this, unpopular networks are at an increased risk, which can especially happen when they're new, or towards the end of their lifetime.
+
+Checkpointing is a proposed mechanism to solve this problem, securing a proof-of-work chain against 51% attacks, even with low mining participation. The main idea is to use a small number of trusted community members, a checkpointing federation, to regularly insert _checkpoints_ into the chain. Checkpoints are special non-proof-of-work blocks that _cannot_ be reverted, essentially confirming all transactions in preceding blocks. Instead of always preferring the longest chain, network nodes should then always adopt the chain with the latest checkpoint. In order to ensure that checkpoint blocks can only be created by the checkpointing federation, the blocks include a list of member signatures signing the preceding block hash. The honest nodes will have to be aware of each members public key in order to verify such blocks. For more details, see especially section 3 of [the paper](https://iohk.io/en/research/library/papers/securing-proof-of-work-ledgers-via-checkpointing/), or [the video discussing it](https://www.youtube.com/watch?v=aasUIB1W81E).
+
+Morpho is an implementation of such a checkpointing federation, based on [OBFT](https://iohk.io/en/research/library/papers/ouroboros-bfta-simple-byzantine-fault-tolerant-consensus-protocol/) to provide consensus and byzantine fault tolerance among the federation members. It is built to interact with [Mantis](https://github.com/input-output-hk/mantis), an Ethereum Classic client that implements support for checkpoint blocks. This scheme has been proposed in [ECIP 1097](https://ecips.ethereumclassic.org/ECIPs/ecip-1097), which has been previously [discussed](https://www.youtube.com/watch?v=dentctKr880), and [demonstrated](https://www.youtube.com/watch?v=1JeL4pAjmXM). A formal specification of Morpho is available [here](https://github.com/input-output-hk/OBFT-Checkpointing-Specification/blob/master/formal-spec/formal-spec.pdf).
```

Comment:
is an implementation of -> implements
A formal specification of Morpho is available [here] -> For more information, see [the formal Morpho specification]